app/controllers/practices_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
   # GET /innovations/1
   # GET /practices/1.json
   def show
+    @allowed_hosts = ['https://dev.marketplace.va.gov', 'https://staging.marketplace.va.gov', 'https://marketplace.va.gov', 'http://localhost:3200']
```

Comment:
I'm not sure we need this line of code. We should just be able to change the condition on the overview page to `.include?('marketplace.va.gov')` and it should work.